dotnet-desktop-guide/net/winforms/controls-design/snippets/designer-properties-overview/csharp/CompassRose.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel;
```

Comment:
Can you serialize/test this with the current 17.8 Preview 2 version (out-of-proc).
The `InitializeComponent` code should be optimized better there.

It's different for In-Proc and out-of-proc.